### PR TITLE
Fix sentry setup by including sentry environment variables in docker-compose.yml 

### PR DESCRIPTION
**Pull request by adiprerepa**
This is the reason sentry isn't working for `dev` and `prod`.

Related: https://github.com/codalab/codalab-worksheets/issues/3416

After this is merged, https://github.com/codalab/deployment/blob/master/ansible/roles/server/vars/main.yml#L11-L12 env vars should work.

### CODE REVIEW

**Comment by adiprerepa** [comment]
once the docker images build, I'll deploy this branch to dev

**Review by epicfaace — APPROVED** [review]
Looks good. Have you been able to test that this PR actually fixes it?

**Comment by adiprerepa** [comment]
was waiting for GHA to finish - will deploy to dev and test

**Review by epicfaace — COMMENTED** [review]
Ok, let me know once you successfully test, and then feel free to merge.

**Comment by adiprerepa** [comment]
So this is odd..

`CODALAB_SENTRY_INGEST_URL` is set to the right URL but `CODALAB_SENTRY_ENVIRONMENT` is just blank, even when hard-coded, no different from `public_hostname` in `dev/main.yml`. no idea why.

File: codalab/worker_manager/azure_batch_worker_manager.py [hunk 1]
```diff
@@ -133,6 +137,9 @@ def start_worker_job(self) -> None:
             task_container_run_options.append(
                 '--env CODALAB_SENTRY_INGEST_URL=%s' % CODALAB_SENTRY_INGEST
             )
+            task_container_run_options.append(
+                '--env CODALAB_SENTRY_ENVIRONMENT=%s' % CODALAB_SENTRY_ENVIRONMENT
```
Comment (adiprerepa) [inline codalab/worker_manager/azure_batch_worker_manager.py:141]: @epicfaace do you think we need this here? Is the sentry sdk initialized in the worker managers?

Comment (epicfaace) [reply codalab/worker_manager/azure_batch_worker_manager.py:141]: I'm not sure, but we should probably make an issue if that's not the case (I believe that nevertheless, these env variables will probably be passed on to the workers, which **do** have the sentry sdk initialized).

**Review by epicfaace — APPROVED** [review]

**Pull request merged**